pkg/config/agent/sysadvisor/qosaware/qos_aware_plugin.go

```diff
@@ -32,15 +32,17 @@ type QoSAwarePluginConfiguration struct {
 	*resource.ResourceAdvisorConfiguration
 	*server.QRMServerConfiguration
 	*reporter.HeadroomReporterConfiguration
+	*reporter.NodeMetricReporterConfiguration
 	*model.ModelConfiguration
 }
 
 // NewQoSAwarePluginConfiguration creates a new qos aware plugin configuration.
 func NewQoSAwarePluginConfiguration() *QoSAwarePluginConfiguration {
 	return &QoSAwarePluginConfiguration{
-		ResourceAdvisorConfiguration:  resource.NewResourceAdvisorConfiguration(),
-		QRMServerConfiguration:        server.NewQRMServerConfiguration(),
-		HeadroomReporterConfiguration: reporter.NewHeadroomReporterConfiguration(),
-		ModelConfiguration:            model.NewModelConfiguration(),
+		ResourceAdvisorConfiguration:    resource.NewResourceAdvisorConfiguration(),
+		QRMServerConfiguration:          server.NewQRMServerConfiguration(),
+		HeadroomReporterConfiguration:   reporter.NewHeadroomReporterConfiguration(),
+		NodeMetricReporterConfiguration: reporter.NewNodeMetricReporterConfiguration(),
```

Comment:
Suggestion: Organize as ReporterConfiguration -> HeadroomReporterConfiguration & NodeMetricReporterConfiguration, to be consistent with ResourceAdvisorConfiguration.

Reply:
Done. Also modified ReporterOptions